client/jetpack-onboarding/steps/summary.jsx

```diff
@@ -18,24 +18,24 @@ import FormattedHeader from 'components/formatted-header';
 import PageViewTracker from 'lib/analytics/page-view-tracker';
 import { getUnconnectedSiteUrl } from 'state/selectors';
 import {
+	JETPACK_ONBOARDING_STEP_TITLES as STEP_TITLES,
 	JETPACK_ONBOARDING_STEPS as STEPS,
 	JETPACK_ONBOARDING_SUMMARY_STEPS as SUMMARY_STEPS,
 } from '../constants';
 
 class JetpackOnboardingSummaryStep extends React.PureComponent {
 	renderCompleted = () => {
-		const stepsCompleted = [
-			SUMMARY_STEPS.SITE_TITLE_DESCRIPTION,
-			SUMMARY_STEPS.SITE_TYPE,
-			SUMMARY_STEPS.HOMEPAGE_TYPE,
-			SUMMARY_STEPS.CONTACT_FORM,
-		];
-		return map( stepsCompleted, ( fieldLabel, fieldIndex ) => (
-			<div key={ fieldIndex } className="steps__summary-entry completed">
-				<Gridicon icon="checkmark" size={ 18 } />
-				{ fieldLabel }
-			</div>
-		) );
+		return map(
+			this.props.steps,
+			stepName =>
+				stepName !== STEPS.SUMMARY && (
```

Comment:
I guess you could `omit( this.props.steps, STEPS.SUMMARY )` two lines above instead.

Reply:
Good call 👍  Done in 7f28b09 (just used `without` because `steps` is an array).